corral/log/log.pony

```diff
@@ -0,0 +1,64 @@
+"""
+# Logging in corral
+
```

Comment:
I think the most important thing for the package docs is to detail the various components and how they interact so...

uses Pony standard logging.

There's a log level formatter that acts as a wrapper around others so that we get INFO etc output without having to spread throughout the codebase.

And here is how it is used with other formatters.